README.md

```diff
@@ -1,73 +1,41 @@
-
-First, you need to add the webpacker-react gem to your Rails app Gemfile:
+First, you need to add this gem to your Rails app Gemfile:
 
 ```ruby
-gem 'webpacker-react', "~> 1.0.0.beta.1"
+gem 'react-components-rails', "~> 1.0.0.beta.1"
 ```
 
 Once done, run `bundle` to install the gem.
 
-Then you need to update your `package.json` file to include the `webpacker-react` NPM module:
+Then you need to update your `package.json` file to include the `react-components-rails` Javascript module:
 
-`./bin/yarn add webpacker-react`
+`yarn add react-components-rails`
 
 You are now all set!
 
 ### Note about versions
 
-Webpacker-React contains two parts: a Javascript module and a Ruby gem. Both of those components respect [semantic versioning](http://semver.org). **When upgrading the gem, you need to upgrade the NPM module to the same minor version**. New patch versions can be released for each of the two independently, so it is ok to have the NPM module at version `A.X.Y` and the gem at version `A.X.Z`, but you should never have a different `A` or `X`.
+React-Components-Rails contains two parts: a Javascript module and a Ruby gem. Both of those components respect [semantic versioning](http://semver.org). **When upgrading the gem, you need to upgrade the NPM module to the same minor version**. New patch versions can be released for each of the two independently, so it is ok to have the NPM module at version `A.X.Y` and the gem at version `A.X.Z`, but you should never have a different `A` or `X`.
 
 ## Usage
 
 The first step is to register your root components (those you want to load from your HTML).
-In your pack file (`app/javascript/packs/*.js`), import your components as well as `webpacker-react` and register them. Considering you have a component in `app/javascript/components/hello.js`:
-
-```javascript
-import Hello from 'components/hello'
-import WebpackerReact from 'webpacker-react'
-
-WebpackerReact.setup({Hello}) // ES6 shorthand for {Hello: Hello}
-```
-
-### With Turbolinks
-
-You have to make sure Turbolinks is loaded before calling `WebpackerReact.initialize()`.
-
-For example:
+In your app entry file, import your components as well as `react-components-rails` and register them. Considering you have a component in `app/javascript/components/hello.js`:
 
 ```javascript
-import Hello from 'components/hello'
-import WebpackerReact from 'webpacker-react'
-import Turbolinks from 'turbolinks'
-
-Turbolinks.start()
-
-WebpackerReact.setup({Hello})
-```
-
-You may also load turbolinks in regular asset pipeline `application.js`:
+import Hello from "components/hello"
+import ReactComponentsRails from "react-components-rails"
 
-```javascript
-//= require "turbolinks"
+ReactComponentsRails.setup({ Hello }) // ES6 shorthand for {Hello: Hello}
```

Comment:
Since turbolink has been deprecated in rails7 in favor of turbo-rails, we have made it work as follows.

However, this method may affect performance.
```suggestion
document.addEventListener("turbo:load", function() {
  ReactComponentsRails.setup({ Hello }) // ES6 shorthand for {Hello: Hello}
})
```